patches/minecraft/net/minecraft/entity/EntityLiving.java.patch

```diff
@@ -39,7 +39,15 @@
      public float func_70603_bj()
      {
          return 1.0F;
-@@ -1003,7 +1015,7 @@
+@@ -991,6 +1003,7 @@
+ 
+     public static EntityEquipmentSlot func_184640_d(ItemStack p_184640_0_)
+     {
++        final EntityEquipmentSlot slot = p_184640_0_.func_77973_b().getEquipmentSlot(p_184640_0_); if (slot != null) return slot; // FORGE: Allow modders to set a non-default equipment slot for a stack; e.g. a non-armor chestplate-slot item
```

Comment:
Seriously THIS IS TWO LINES OF CODE
Put it on two lines!
Jesus the point of the 'Keep patch files small' is to MAKE THEM READABLE.
To prevent people from reformatting vanilla code because they feel like it. 
So we can see FUNCTIONAL changes vs formatting changes.
If you're just adding a hunk of code, IT DOESN'T need to be all in one line!